setup.py

```diff
@@ -9,6 +9,7 @@
     long_description = fh.read()
 
 requirements = [
+    "setuptools",
```

Comment:
Is this required at install time? I know we need it for build, but I'm not sure if it will be required for the runtime.